src/scripting/zscript/zcc_compile.cpp

```diff
@@ -404,6 +404,8 @@ ZCCCompiler::ZCCCompiler(ZCC_AST &ast, DObject *_outer, PSymbolTable &_symbols,
 					}
 				}
 				break;
+			default:
+				break;
```

Comment:
`59 enumeration values not handled in switch: 'AST_Identifier', 'AST_Class', 'AST_Struct'...` warning is still reported for inner `switch` statement.

Honestly, by looking closely at this code, I'm not sure that it's correct in the first place. Have no idea what's the point of the inner `switch` at all.